package.json

```diff
@@ -437,11 +437,28 @@
                   "debug",
                   "remote",
                   "test",
-                  "exec"
+                  "exec",
+                  "replay",
+                  "core"
                 ],
-                "description": "One of 'auto', 'debug', 'remote', 'test', 'exec'.",
+                "description": "One of 'auto', 'debug', 'remote', 'test', 'exec', replay'.",
                 "default": "auto"
               },
+              "executable": {
+                "type": "string",
+                "description": "Compiled program to be targeted in replay in required scenarios",
+                "default": ""
+              },
+              "traceDirectory": {
+                "type": "string",
+                "description": "Directory in which the record trace is located. For use on 'replay' mode only",
+                "default": ""
+              },
+              "coreDumpDirectory": {
```

Comment:
Is there a reason to work with the directory, instead of the file?
`dlv core` takes the core file, right?

Reply:
Yeah, the command is `dlv core <executable> <coredump> [args]`

And you're right, it is a misnomer indeed, it should be coreDumpPath, gonna fix it